src/seagulls/examples/_game_state.py

```diff
@@ -0,0 +1,7 @@
+from seagulls.engine import IGameScene
+
+
+class GameState:
+    active_scene: IGameScene = None
```

Comment:
the proper way to define the type for something that is optional (can be `None`) is with `Optional[]` so this would turn into:

```python
class GameState:
    active_scene: Optional[IGameScene] = None
```

Reply:
Thanks! Made the change.